File: Ciência-da-Computação/INTRODUCAO-A-PYTHON/dia-1-introducao-a-python/exercise_6.py
def create_triangle(side_a, side_b, side_c):
    if side_a == side_b == side_c:
        print("Triângulo Equilátero")
    elif (
        side_a == side_b != side_c
        or side_a == side_c != side_b
        or side_b == side_c != side_a
    ):
        print("Triângulo Isósceles")
    else:
        print("Triângulo Escaleno")


create_triangle(3, 4, 5)

File: Ciência-da-Computação/INTRODUCAO-A-PYTHON/dia-1-introducao-a-python/exercise_6.js
const createTriangle = (sideA,sideB,sideC) => {
    if(sideA === sideB && sideB === sideC) return console.log('Triâgulo Equilátero');
    if(
        (sideA === sideB && sideA !== sideC)
        || (sideA === sideC && sideA !== sideB)
        || (sideC === sideB && sideC !== sideA)) {
            return console.log('Triângulo Isósceles');
        }
    return console.log('Triângulo Escaleno');
}

createTriangle(3,4,5);